pkg/katautils/config_test.go

```diff
@@ -1491,6 +1491,53 @@ func TestDefaultMachineAccelerators(t *testing.T) {
 	assert.Equal(machineAccelerators, h.machineAccelerators())
 }
 
+func TestDefaultCpuFeatures(t *testing.T) {
+	assert := assert.New(t)
+	cpuFeatures := "abc,123,rgb"
```

Comment:
Since some config options are space-separated, it may be a good idea to add a test to ensure that if the user mis-specifies the feature as a space-separated list (`abc 123 rgb`), this results in a clear error.

Reply:
ok

Reply:
hi @jodh-intel. Could you please give me an example of space-separated config options? I searched qemu user doc, seems no such option.
Currently after this patch set, "abc 123 rgb" will be passed as "-cpu host,abc 123 rgb". So definitely the qemu can't boot up the guest.

Reply:
Hi @justin-he - sorry, I meant space-separated options in `configuration.toml`. The only one is in fact `kernel_params`, but I just wondered what would happen if the user inadvertently specified the cpu features in `configuration.toml` as a space-separated list. If qemu gives an obvious error, I guess we can let that scenario "pass through" to qemu. But if you get a really obscure "failed to boot" message, it might be helpful if we can detect it in the runtime and error. If the latter, we could add a unit test. It's just a thought though and not a blocker.